IFC4x3/Constants/c/COMMUNICATIONTERMINAL_1C$9BNNcj9mx0W2dT2QsBb.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="COMMUNICATIONTERMINAL_1C9BNNcj9mx0W2dT2QsBb" Name="COMMUNICATIONTERMINAL" UniqueId="4cfc92d7-5e6b-49c3-b020-0a77426b62e5">
+	<Documentation>A communication terminal is an audio communication device that usually installed along the railway in order to be used by general public or railway agents for communication. It may specifically be used to make calls to emergency services in tunnels.</Documentation>
```

Comment:
```suggestion
	<Documentation>A communication terminal is an audio communication device that is usually installed along transportation infrastructure (railways, roads, tunnels etc.) in order to be used by the general public or operation agents for communication. It may specifically be used to make calls to emergency services in tunnels.</Documentation>
```

Reply:
Name has been generalised, but text is still railway specific, suggestion means semantics allow it to be used for comm terminals on roads or in tunnels also.